app/models/item.rb

```diff
@@ -4,6 +4,10 @@ class Item < ApplicationRecord
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
 
+  validates :name, presence: { message: "Please add a Name for the item." }
```

Comment:
The only thing I'd say could be refactored in the future would be to actually use these message in the flash message so the user knows what is specifically wrong with the form.